Automattic/simplenote-macos
# Tags List: Adjusting Separator

### Fix
In this PR we're adjusting the Header Separator in the Tags List Area. 
This was spotted by @eshurakov in #833.

cc @SylvesterWilmott 
cc @eshurakov 

Thank you gentleman!!

### Test
1. Log into your account
2. Scroll over the Tags List so that the Header separator shows up

- [ ] Verify the Tags separator looks super in Light Mode
- [ ] Verify the Tags separator looks super in Dark Mode as well!

### Release
These changes do not require release notes.

### Screenshots
New Dark|New Light
-|-
<img width="300" alt="Screen Shot 2021-02-08 at 4 38 38 PM" src="https://user-images.githubusercontent.com/1195260/107272462-7fca4600-6a2c-11eb-9328-f0c1b4e55bab.png">|<img width="300" alt="Screen Shot 2021-02-08 at 4 38 58 PM" src="https://user-images.githubusercontent.com/1195260/107272476-83f66380-6a2c-11eb-9009-5c103abead2e.png">
